dev/dll/Microsoft.UI.Xaml.vcxproj.filters

```diff
@@ -147,6 +147,81 @@
     <Page Include="@(PageRequiringCustomCompilation)" />
     <Page Include="@(PageRequiringCustomCompilation)" />
     <Page Include="@(PageRequiringCustomCompilation)" />
+    <Page Include="@(PageRequiringCustomCompilation)" />
```

Comment:
Uhm what is happening here?

Reply:
These are benign but weird. @llongley might have some idea on what might be causing it.